.github/workflows/integration-test.yml

```diff
@@ -5,8 +5,6 @@ on:
     branches:
       - main
   pull_request:
-    branches:
```

Comment:
I removed this so that the pipelines will run when we open a PR from one branch to another that's not `main`.